apps/src/maze/CollectorGemCounter.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import color from '../util/color';
+import msg from './locale';
+
+const styles = {
+  container: {
+    display: 'inline-block',
+    fontFamily: '"Gotham 5r"',
+    paddingRight: 10,
+    pointerEvents: "none",
+    textAlign: 'right',
+    verticalAlign: 'top',
+  },
+  label: {
+    color: color.lighter_gray,
+    textTransform: 'capitalize'
+  },
+  hr: {
+    borderColor: color.lighter_gray,
+    borderWidth: 2,
+    margin: '0 auto',
+  },
+  gemImage: {
+    backgroundSize: "100%",
+    display: "inline",
+    padding: 5
+  },
+  checkmark: {
+    color: 'lightgreen',
+    fontSize: 18,
+    left: 5,
+    lineHeight: '25px',
+    position: 'relative',
+    top: 7,
+  },
+};
+
+export class CollectorGemCounter extends React.Component {
```

Comment:
I originally had one, but had to remove it; Storybook doesn't play nicely with app-specific i18n ('maze/locale') unless we do [special override work](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/webpack.js#L24-L28) and that seemed like too much of a refactor to justify